documents/Code of Ethics.md

```diff
@@ -1,87 +1,97 @@
-      2. Delegates should work alongside the organizer(s) to plan and resolve any of these reasons listed under 3.3.1.
-   4. Delegates and Organizers should only be officially listed for a competition if they perform a necessary role in the running of the event.
+         5. Inexperience of the Organizer
+         6. Past problems with an event held by the Organizer/community
+      2. Delegates should work alongside the Organizer(s) to plan and resolve any of these reasons listed under 3.3.1.
+   4. Delegates and Organizers should only be officially listed for a competition if they perform a significant role.
+      1. Delegates and Organizers should agree on their roles for the competition before the competition is announced.
 4. **Conflicts of Interest**
-   1. WCA Staff must report all potential conflicts of interest that may arise and affect their WCA Staff role. Conflicts of interest will be reported to the WEC. These include but are not limited to:
+   1. WCA Staff must report all potential conflicts of interest that may arise and affect their WCA Staff role to the WEC using [this form]{https://docs.google.com/forms/d/e/1FAIpQLSca81GIwjguJoWrPcbgabkRpdgJqbusIf9RBR7ObNNNL9kvqw/viewform?usp=sf_link} . These include, but are not limited to:
       1. Sponsorships
-      2. A stake in businesses trading within the community
+      2. A stake in businesses trading within the cubing community.
          1. Example: Being the owner of a cubing store/business.
-      3. Any personal profit gained by actions taken within their WCA Staff role
    2. WCA Staff must not receive sponsorship(s) for their role within the WCA.
-      1. Any sponsorship details of a WCA Staff member must be reported to the WEC.
-   3. WCA Staff must not promote a third party brand who are not the competition sponsor(s) or sponsors of the WCA, within their role in the WCA. This includes but is not limited to:
-      1. Wearing the apparel of a sponsor while acting within their WCA Staff role at a competition
-      2. Promotions in emails to competitors
-      3. References in the email signature block
-   4. WCA Delegates must not wear sponsorship apparel of a personal sponsor during a competition in which they are listed as a Delegate.
-      1. Exception: Listed Delegates may only wear their sponsorship apparel while they are competing at a competition.
+      1. Any sponsorship details of a WCA Staff member as a competitor must be reported to the WEC.
+   3. WCA Staff must not use their role to promote a third party brand who are not [a WCA-recognized regional organization](https://www.worldcubeassociation.org/organizations), the competition sponsor(s), or sponsors of the WCA. This includes but is not limited to:
+      1. Promotions in emails to competitors
+      2. References in the email signature block
+   4. WCA Delegates must not wear apparel which could reasonably be interpreted as an official affiliation or agreement with the WCA.

```

Comment:
Is it possible to make this more specific? I feel like this would prevent unlisted Delegates at a competition like Worlds from wearing what they wanted at any time.

Reply:
It was changed to be worded in a way that moves more towards an explanation of what we are trying to acheive with it being in the CoE. As opposed to nailing down every specific instance like the regulations, which would be impossible for us to do anyway because of WEC's scope.

This would prevent unlisted Delegates from wearing sponsored garments while not competing, correct. However this closes the loophole you pointed out before about unlisteds becoming listed causing issues if they were promoting a third party during that competition. That instance can occur at more competitions than the biennial World or Continental championships, so on balance we decided that the same rule for all people holding a Delegate position was fairer.